build/buildConfig.js

```diff
@@ -142,31 +153,19 @@ module.exports = (bundles, themeEntries, paths, extractThemesPlugin, prod, publi
                 }] // inline base64 URLs for <=8k images, direct URLs for the rest
             },
             {
-                test: /\.jsx$/,
-                exclude: /(ol\.js)$|(Cesium\.js)$/,
-                use: [{
-                    loader: "react-hot-loader"
-                }],
-                include: paths.code
-            }, {
                 test: /\.jsx?$/,
                 exclude: /(ol\.js)$|(Cesium\.js)$/,
                 use: [{
-                    loader: "babel-loader"
+                    loader: "babel-loader",
+                    options: {
+                        configFile: path.join(__dirname, 'babel.config.js')
+                    }
                 }],
-                include: [
-                    paths.code,
-                    // the followings are here to skip issues caused by uglifyPlugin (see #4212)
-                    // TODO: a solution to this issue that uses only libs
-                    // like https://www.npmjs.com/package/uglify-js-es6#harmony when stable, terser, babel-minify ...
-                    path.join(paths.base, "node_modules", "query-string"),
-                    path.join(paths.base, "node_modules", "strict-uri-encode"),
-                    path.join(paths.base, "node_modules", "split-on-first")
-                ]
+                include: [paths.code, path.join(__dirname, "..", "node_modules", "query-string"), path.join(__dirname, "..", "node_modules", "strict-uri-encode"), path.join(__dirname, "..", "node_modules", "split-on-first")]
```

Comment:
@mbarto Here you can use the fix made by @offtherailz  [here](https://github.com/geosolutions-it/MapStore2/pull/4268/files)

by replacing __dirname, "..",  with paths.base

Please would you mind to list them vertically? (big thanks)

Reply:
as you may know, the best way to make this happen is a PR